zero_bin/worker/src/main.rs

```diff
@@ -38,6 +38,7 @@ async fn main() -> Result<()> {
 
     dotenv().ok();
     init::tracing();
+    set_circuit_cache_dir_env_if_not_set()?;
```

Comment:
I think we'd also need to call this in `verifier/src/main.rs`?

Reply:
Right, thanks!